src/functionalTest/groovy/com/autonomousapps/android/AdviceSpec.groovy

```diff
@@ -32,6 +32,9 @@ final class AdviceSpec extends AbstractAndroidSpec {
           onUnusedAnnotationProcessors {
               fail()
           }
+          onCompileOnly {
+              fail()
```

Comment:
this is missing a parameter value for "ignoring" some compileOnly dep, to fully test the new extension.